src/layer/Popup.js

```diff
@@ -97,6 +97,10 @@ export var Popup = DivOverlay.extend({
 		// Controls the presence of a close button in the popup.
 		closeButton: true,
 
+		// @option closeButtonAriaLabel: String = 'Close popup'
+		// Specifies the 'aria-label' attribute of the close button.
+		closeButtonAriaLabel: 'Close popup',

```

Comment:
Can we rename this to `closeButtonLabel`? ARIA is a technical detail, the important thing is that there is an accessible label.

Reply:
Also not sure if "Button" should be in the name, the zoom controls don't use it:

<html>
<body>
<!--StartFragment-->

zoomInText | String | &lt;span aria-hidden="true">+&lt;/span> | The text set on the 'zoom in' button.
-- | -- | -- | --
zoomInTitle | String | 'Zoom in' | The title set on the 'zoom in' button.
zoomOutText | String | &lt;span aria-hidden="true">&#x2212;&lt;/span> | The text set on the 'zoom out' button.
zoomOutTitle | String | 'Zoom out' | The title set on the 'zoom out' button.

<!--EndFragment-->
</body>
</html>

And maybe the name should be more descriptive about which close button it is, for consistency with the zoom controls? `PopupCloseLabel` perhaps?

Reply:
I think I'd still prefer `closeButtonLabel` as there is already an option that is called `closeButton`, which controls the visibility of the button itself. Given the context is initializing the `Popup` I would argue that is clear enough.

Reply:
That should be all as far as I know. Thanks @ShivangMishra